S/SPRAL/build_tarballs.jl

```diff
@@ -3,37 +3,44 @@
 using BinaryBuilder, Pkg
 
 name = "SPRAL"
-version = v"0.1.0"
+version = v"2023.03.29"
 
 # Collection of sources required to complete build
 sources = [
-    GitSource("https://github.com/ralna/spral.git", "cd2d2e817275f16d586cf72767631b3c7472ce02")
+    GitSource("https://github.com/ralna/spral.git", "b1d6a7409d41d575bff7106a4e9d9a60b8935c2e")
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/spral
+
+if [[ "${target}" == *mingw* ]]; then
+  BLAS_LAPACK="-L${libdir} -lblastrampoline-5"
```

Comment:
Aren't you duplicating `-L${libdir}` everywhere?

Reply:
Indeed, thanks Mosè!